web/styles/compose.css

```diff
@@ -1137,10 +1137,10 @@ textarea.new_message_textarea {
     }
```

Comment:
`enter_sends_major` seems like it needs to scale too?

Reply:
Oops, updated that, and about to update screenshots.

Reply:
Updated :)